File: dotnet-back-end/ktdi-app/Model/RoomRegistration.cs
using Microsoft.AspNetCore.Authorization;
using Microsoft.AspNetCore.Mvc;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

[Authorize]
[ApiController]
[Route("api/[controller]")]
public class RoomRegistration
{
    [Key]
    [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
    public int RoomId { get; set; }

    public string RoomType { get; set; }

    public string Block { get; set; }

    public int Level { get; set; }

    public int Number { get; set; }

    public string Status { get; set; } = "available";
}

File: springboot-back-end/src/main/java/com/model/RoomRegistration.java
package com.model;

import jakarta.persistence.*;
import lombok.Data;

@Entity
@Table(name = "RoomRegistration")
@Data
public class RoomRegistration {

    @Id
    @GeneratedValue
    @Column(name = "room_id")
    private Long roomId;

    @Column(name = "room_type")
    private String roomType;
    private String block;
    private String level;
    private String number;
    private String status = "available";
}